core/components/atoms/text/text.js

```diff
@@ -4,41 +4,46 @@ import styled from 'styled-components'
 
 import { colors, fonts } from '@auth0/cosmos-tokens'
 
-const StyledTextSubdued = styled.span`
+const Text = props => {
+  if (props.type === 'subdued') {
+    return <Text.Subdued {...props}>{props.children}</Text.Subdued>
+  }
+  if (props.type === 'allcaps') {
+    return <Text.AllCaps {...props}>{props.children}</Text.AllCaps>
+  }
+  if (props.type === 'strong') {
+    return <Text.Strong {...props}>{props.children}</Text.Strong>
+  }
+  return props.children
+}
+
+Text.Subdued = styled.span`
```

Comment:
Not sure if this is the right thing to do - These are not children of `Text`. These are different types of `Text`

@francocorreasosa What do you think?

Reply:
I thought that too, but I think this is still a nice way to organize them (and avoid having to export all different kind of text we have now and in the future).